server/apps/archive/common.py

```diff
@@ -378,6 +378,9 @@ def item_schema(extra=None):
             'type': 'string',
             'mapping': not_analyzed,
             'nullable': True
+        },
+        SEQUENCE: {
```

Comment:
Discussed with Mayur, this will affect existing packages so we will handle this during a refactor where we need to wipe the db.